zebrad/tests/acceptance.rs

```diff
@@ -1473,3 +1487,413 @@ where
 
     Ok(())
 }
+
+/// Type alias for a boxed state service.
+type BoxStateService =
+    BoxService<zebra_state::Request, zebra_state::Response, zebra_state::BoxError>;
+
+/// Type alias for the RPC client to communicate with a lightwalletd instance.
+type LightwalletdRpcClient = lightwalletd::rpc::compact_tx_streamer_client::CompactTxStreamerClient<
+    tonic::transport::Channel,
+>;
+
+/// Optional environment variable with the cached state for lightwalletd.
+///
+/// Can be used to speed up the [`sending_transactions_using_lightwalletd`] test, by allowing the
+/// test to reuse the cached lightwalletd synchronization data.
+const LIGHTWALLETD_DATA_DIR_VAR: &str = "LIGHTWALLETD_DATA_DIR";
```

Comment:
Minor: Can we make the `ZEBRA_CACHED_STATE_PATH` and `LIGHTWALLETD_DATA_DIR` names consistent?

Reply:
The existing "run lightwalletd tests" env var is called `ZEBRA_TEST_LIGHTWALLETD`, if that helps.

Reply:
I just ended up using these as-is in #3511, we can change them later if you want.

Reply:
Done.